tests/helpers.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python3
+# Copyright 2021 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""Helper functions for writing tests."""
+
+from typing import Callable, Dict
```

Comment:
`Dict` can be removed from imports as it's not used in the code (and is deprecated in Python 3.9, which should be shipped by jammy - good to remove if we support jammy in the future).

Reply:
My bad, thanks for catching!